librdmacm/examples/rstream.c

```diff
@@ -491,6 +491,12 @@ static int client_connect(void)
 		rdma_freeaddrinfo(rai);
 	else
 		freeaddrinfo(ai);
+	if (src_addr) {
```

Comment:
This whole bit seems clearer if it sjust

if (rai)
   rdma_freeaddrinfo(rai)
if (ai)
   freeaddrinfo(ai)
if  (rai_src)
   rdma_freeaddrinfo(rai_src)

etc